index.js

```diff
@@ -1,253 +1,38 @@
 
-var fs = require('fs'),
-    levelup = require('levelup'),
-    multilevel = require('level-multiply'),
-    parser = require('osm-pbf-parser')(),
-    through = require('through2'),
-    suggester = require('pelias-suggester-pipeline'),
-    propStream = require('prop-stream'),
-    settings = require('pelias-config').generate(),
-    dbclient = require('pelias-dbclient')(),
-    OsmiumStream = require('osmium-stream');
-
-var Backend = require('geopipes-elasticsearch-backend');
-
-// esclient.livestats();
+/**
+  The openstreetmap importer provides a way of parsing, mapping and
+  augmenting OSM data in to elasticsearch.
+**/
 
-var backend = function( index, type ){
-  return new Backend( dbclient.client, index, type );
+var elasticsearch = require('pelias-dbclient'),
+    adminLookup = require('pelias-admin-lookup'),
+    suggester = require('pelias-suggester-pipeline'),
+    dbmapper = require('./stream/dbmapper');
```

Comment:
I think the database mapper stream should be moved into `pelias-dbclient`, since we're currently repeating that same bit of code in all of our importers. If we're still keen on keeping that package generalized for anyone's use, we should then re-publish it as an `elasticsearch-bulk-indexer` and rewrite `pelias-dbclient` as a lightweight wrapper for it.